renovate.json

```
@@ -15,12 +15,10 @@
       ],
       "matchDatasources": ["docker"],
       "matchPackageNames": [
-        "registry.access.redhat.com/ubi9/ubi",
         "registry.access.redhat.com/ubi9/ubi-minimal"
       ],
       "enabled": true,
       "versioning": "redhat",
```

```suggestion
      "versioning": "docker",
```

Similar to the [external-dns PR](https://github.com/openshift/external-dns/pull/114#discussion_r2564219363). If we intend to keep the automated image bump PRs coming for the `latest@sha`, this should be updated to `docker` versioning I think.